README.md

```diff
@@ -22,12 +22,20 @@ Unlike the Yocto BMAP tool, `bmap-writer` is C++ based does not require Python a
 1. Create a BMAP file for your disk image using `bmaptool`.
 2. Use `bmap-writer` to write the image to your target device, specifying the BMAP file for efficient block mapping.
 
-## Example Usage
+## Usage
 
 ```
-bmap-writer <image-file> <bmap-file> <target-device>
+bmap-writer [-b <max-buf-size>] [-v] <image-file> <bmap-file> <target-device>
 ```
 
+The `<max-buf-size>` parameter can be used to limit the write buffer size on systems with limited RAM.
+It can be specified as a plain number or using the `K` (KiB), `M` (MiB), `G` (GiB) or `T` (TiB) suffixes.
+If not specified, the write buffer size will be variable and will depend on the size of the ranges specified inside the BAMP file.
+
+If `-n` is specified, the checksum verification is skipped.
```

Comment:
same here can we keep the checksum verification enforced ?

Reply:
I introduced the `n` option to mimic the feature provided by the original bmatools, but it may be removed. It's a non-mandatory option though, if not specified the default behavior will be kept.

Reply:
I removed the skip option entirely in the updated MR.